### PR TITLE
fix(lockfile): adapt message to remove TODO occurrence

### LINKED ISSUE #412: Remove "TODO" comment from prisma lock file
The migrate lockfile contains a "TODO" comment, but since this will be committed to VCS IDEs may show these TODOs to the user. IntelliJ also warns about these TODOs, even though they are irrelevant for the user. We should remove the "TODO" occurrence.

telegram-cloud-photo-size-2-5222071807325089081-y

telegram-cloud-photo-size-2-5222071807325089082-y

### PR DESCRIPTION
**Pull request by steebchen**
fix #412

### CODE REVIEW

File: src/utils/LockFile.ts [hunk 1]
```diff
@@ -1,9 +1,9 @@
 import { LockFile } from '../types'
 
+// TODO add a link to documentation
 const header = `# IF THERE'S A GIT CONFLICT IN THIS FILE, DON'T SOLVE IT MANUALLY!
 # INSTEAD EXECUTE \`prisma migrate fix\`
```
Comment (janpio) [inline src/utils/LockFile.ts:5]: Uh, this should not exist any more I think. @timsuchanek ?

Comment (steebchen) [reply src/utils/LockFile.ts:5]: You mean the "fix" command? Well then maybe we could just change it to:

```
# Prisma Migrate lockfile v1
```

Comment (janpio) [reply src/utils/LockFile.ts:5]: Yes, Tim and me had a conversation about this recently and also from PRs I was under the impression that this was already removed 🤷

Comment (janpio) [reply src/utils/LockFile.ts:5]: ping @timsuchanek @Jolg42

Comment (Jolg42) [reply src/utils/LockFile.ts:5]: I will do a cleanup, it looks like it's still there!

**Pull request closed without merging**